ui/admin/ember-cli-build.js

```diff
@@ -36,10 +38,16 @@ module.exports = function (defaults) {
         },
       },
     },
-    // TODO: Update to 4.12 when deprecations are resolved
-    //   as multiple things break when forcing compatibility on > 4.6
-    emberData: {
-      compatWith: '4.6',
+  });
+
+  // TODO: This is config can be removed in ember-data 6.0.
```

Comment:
None blocking comment: should we delete the "is"? so This config can be removed....

Reply:
oh yup great catch!